crates/node/src/catchup.rs

```diff
@@ -193,24 +86,28 @@ impl Node {
         info!(%context_id, %peer_id, "Interval triggered catchup successfully finished");
     }
 
+    // TODO: Is this even needed now? Can it be removed? In theory, a sync will
+    // TODO: take place so long as there is a comparison - i.e. it will send
+    // TODO: everything back and forth until everything matches. But, for e.g. a
+    // TODO: first-time sync, that would be slower than just sending everything
+    // TODO: all at once. So... could this be utilised for that?
```

Comment:
@miraclx just want to make sure this wasn't missed.

Reply:
Yeah, I think this makes sense to retain.

> In theory, a sync will take place so long as there is a comparison

this is what initiates that comparison, right? that's what it did in the transactional model anyway